### PR TITLE
Added metadata to text.py

### PR DESCRIPTION
**Pull request by Azaya89**
I added metadata to examples/plotting/text.py

- [x] issues: fixes #11765 
- [x] tests added / passed

### CODE REVIEW

File: examples/plotting/text.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
+''' A grid plot using descriptive test. This example demonstrates
```
Comment (mosc9575) [inline examples/plotting/text.py]:
```suggestion
''' A grid plot using descriptive text. This example demonstrates
```

Comment (Azaya89) [reply examples/plotting/text.py]: Changes executed.

File: examples/plotting/text.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
+''' A grid plot using descriptive text. This example demonstrates
+how to align text in a plain background in two dimensions. It also
+demonstrates how to change the angle of the texts in the plot.
```
Comment (bryevdv) [inline examples/plotting/text.py:3]: small edit suggestion:
```suggestion
demonstrates how to change the angle of text in a plot.
```

File: examples/plotting/text.py [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.layouts.gridplot
+    :refs: :ref:`ug_basic_layouts_gridplot`
+    :keywords: Slider, gridplot, line, text
```
Comment (bryevdv) [inline examples/plotting/text.py:8]: AFAIK we have not capitalized keywords anywhere else 
```suggestion
    :keywords: slider, gridplot, line, text
```

File: examples/plotting/text.py [hunk 4]
```diff
@@ -1,3 +1,13 @@
+''' A grid plot using descriptive text. This example demonstrates
+how to align text in a plain background in two dimensions. It also
+demonstrates how to change the angle of the texts in the plot.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.layouts.gridplot
+    :refs: :ref:`ug_basic_layouts_gridplot`
```
Comment (bryevdv) [inline examples/plotting/text.py:7]: This section also seems relevant:
```suggestion
    :refs: :ref:`ug_basic_layouts_gridplot`, :ref:`ug_styling_text_properties`
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @Azaya89 just a few comments

**Comment by bryevdv** [comment]
Hi @Azaya89 unfortunately I had not noticed but there is already an earlier PR open to add metadata to this particular example #12990 accordingly I will close this later PR. Fortunately there a lots of examples that can use your attention!

**Pull request closed without merging**